Dockerfiles/agent/install.ps1

```diff
@@ -1,6 +1,28 @@
 $ErrorActionPreference = "Stop"
 Trap { Write-Host "Error in install.ps1: $_" }
 
+function Install-Service {
+  param(
+    [Parameter(Mandatory=$true)][string] $SvcName,
+    [Parameter(Mandatory=$true)][string] $BinPath,
+    [Parameter(Mandatory=$false)][string[]] $Depends
+  )
+  if ( $Depends.count -gt 0 ){
+      New-Service -Name $SvcName -StartupType Manual -BinaryPathName $BinPath -Depends $Depends
+  } else {
+      New-Service -Name $SvcName -StartupType Manual -BinaryPathName $BinPath
+  }
+  #New-EventLog -Source $SvcName -LogName Application -MessageResourceFile $BinPath -CategoryResourceFile $BinPath

```

Comment:
@derekwbrown do you need this commented out line for reference or should we drop it?